lib/v2/uber/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'uber.com': {
+        _name: 'Uber Blog',
+        www: [
+            {
+                title: 'Uber Blog - Engineering',
```

Comment:
```suggestion
                title: 'Engineering',
```
No need to repeat `_name`